packages/core-mobile/app/store/account/listeners.ts

```diff
@@ -64,6 +65,17 @@ const initAccounts = async (
     }
   }
 
+  if (activeWallet.type === WalletType.KEYSTONE) {
+    try {
+      await KeystoneService.refreshPublicKeys()
```

Comment:
nit: looking at the code, it looks like the refreshPublicKeys method is actually just about saving the walletInfo. would be great if you could rename the method and adjust the logger.error message.